rif/relay/gas-costs.md

```diff
@@ -17,9 +17,11 @@ V2 introduces a cheaper template ([SmartWallet](https://github.com/rsksmart/rif-
 
 | RIF Version | SW Template       | Avg. overhead gas |
 |-------------|-------------------|-------------------|
-| 1           | SmartWallet       | 172400            |
-| 2           | CustomSmartWallet | 98070             |
-| 2           | SmartWallet       | 97695             |
+| 0.1           | SmartWallet       | 172400            |
+| 0.2           | CustomSmartWallet | 98070             |
+| 0.2           | SmartWallet       | 97695             |
+| 1             | CustomSmartWallet | TBD               |
+| 1             | SmartWallet       | TBD               |
```

Comment:
I guess the overhead is related to the deployment, but from the text itself it's not clear to me what it measures exactly.

Reply:
To be honest with you, I did not do much changes here, since we talk about not doing any change here. but This is referring to the deployment process. But I added a small explanation.